README.md

```diff
@@ -1,8 +1,27 @@
-Confluent's Apache Kafka client for Golang
-==========================================
+Confluent's Golang Client for Apache Kafka<sup>TM</sup>
+=====================================================
 
-Confluent's Kafka client for Golang wraps the librdkafka C library, providing
-full Kafka protocol support with great performance and reliability.
+**confluent-kafka-go** is Confluent's Golang client for [Apache Kafka](http://kafka.apache.org/) and the
+[Confluent Platform](https://www.confluent.io/product/).
+
+Features:
+
+- **High performance** - confluent-kafka-go is a lightweight wrapper around
+[librdkafka](https://github.com/edenhill/librdkafka), a finely tuned C
+client.
+
+- **Reliability** - There are a lot of details to get right when writing an Apache Kafka
+client. We get them right in one place (librdkafka) and leverage this work
+across all of our clients (also [confluent-kafka-python](https://github.com/confluentinc/confluent-kafka-python)
```

Comment:
Perhaps we can get a clients page on the main website that we can link to? Would make it less maintenance across the repos to keep these lists up to date (also, eventually they may become longer than we want to have included in each repo). The direct links here don't seem all that valuable since most people are looking for a specific language.